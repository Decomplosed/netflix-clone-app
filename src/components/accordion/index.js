import React from 'react';

export default function Accordion({ children, ...restProps }) {
  return <Container></Container>;
}
