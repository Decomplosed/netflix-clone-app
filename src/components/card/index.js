import React from 'react';

export const FeatureContext = createContext();
