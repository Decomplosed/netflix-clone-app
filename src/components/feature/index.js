import React from 'react';
import { Container, Title, SubTitle } from './styles/feature';

export default function Feature({ children, ...restProps }) {
  return <Container></Container>
}
