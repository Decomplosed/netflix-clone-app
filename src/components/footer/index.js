import React from 'react';

export default function Footer({ children, ...restProps }) {
  return <Container {...restProps}></Container>;
}
