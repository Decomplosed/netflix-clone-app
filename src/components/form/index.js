import React from 'react';
import {
  Container,
  Base,
  Error,
  Title,
  Text,
  TextSmall,
  Link,
  Input,
  Submit,
} from './styles/form';

export default function Form({ children, ...restProps }) {
  return <Container {...restProps}>{children}</Container>;
}

Form.Base = function FormBase({ children, ...restProps }) {
  return <Base {...restProps}>{children}</Base>;
};

Form.Error = function FormError({ children, ...restProps }) {
  return <Error {...restProps}>{children}</Error>;
};

Form.Title = function FormTitle({ children, ...restProps }) {
  return <Title {...restProps}>{children}</Title>;
};
