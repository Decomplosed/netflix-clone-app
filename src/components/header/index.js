import React from 'react';

export default Header({bg = true, children, ...restProps}) {}