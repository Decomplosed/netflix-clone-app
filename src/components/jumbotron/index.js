import React from 'react';

export default Jumbotron = () => {
  return <p>I am jumbotron</p>;
};
