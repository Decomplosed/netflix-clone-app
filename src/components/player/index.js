import React, { useState, useContext, createContext } from 'react';
import ReactDOM from 'react-dom';
