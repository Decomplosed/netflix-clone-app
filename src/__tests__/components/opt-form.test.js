import React from 'react';
import { render } from '@testing-library/react';
import { OptForm } from '../../components';

describe('<OptForm />', () => {
  it('Renders the <OptForm /> with populated data', () => {
    const { container, getByText, getByPlaceholderText } = render(
      <OptForm></OptForm>
    );
  });
});
