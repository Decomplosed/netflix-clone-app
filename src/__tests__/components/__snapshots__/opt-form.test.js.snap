// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OptForm /> Renders the <OptForm /> with populated data 1`] = `
<div
  class="opt-form__Container-fo1ax-0 flLcgP"
>
  <input
    class="opt-form__Input-fo1ax-1 sefmy"
    placeholder="Email address"
  />
  <button
    class="opt-form__Button-fo1ax-3 gGweZS"
  >
    Try it now
     
    <img
      alt="Try now!"
      src="/images/icons/chevron-right.png"
    />
  </button>
  <div
    class="opt-form__Break-fo1ax-2 gbGjGL"
  />
  <p
    class="opt-form__Text-fo1ax-4 gTCrYW"
  >
    Ready to watch? Enter your email to create or restart your membership.
  </p>
</div>
`;
